src/scrabble.js

```diff
@@ -1,5 +1,44 @@
-function scrabble() {
+const letternum = {
+  a: 1,
+  e: 1,
+  i: 1,
+  o: 1,
+  u: 1,
+  l: 1,
+  n: 1,
+  r: 1,
+  s: 1,
+  t: 1,
+  d: 2,
+  g: 2,
+  b: 3,
+  c: 3,
+  m: 3,
+  p: 3,
+  f: 4,
+  h: 4,
+  v: 4,
+  w: 4,
+  y: 4,
+  k: 5,
+  j: 8,
+  x: 8,
+  q: 10,
+  z: 10
+}
+
+function scrabble(word) {
+  if (word === null || word === '') return 0
+
+  let sum = 0
+
+  const upperCaseScore = word.toLowerCase().split('')
```

Comment:
Would change upperCaseScore to lowerCaseScore. Or maybe even lowerCaseArray to be really explicit. Otherwise awesome :)